scripts/src/public/services/orders.ts

```diff
@@ -162,9 +162,10 @@ export async function createExternalOrder(jwt: string, user: User, logger: Logge
 	const payload = await validateExternalOrderJWT(jwt, user.appUserId, logger);
 
 	let order = await db.Order.findOne({ userId: user.id, offerId: payload.offer.id });
+	const completed_orders = await db.Order.getCompletedOrder( user.id, payload.offer.id);
```

Comment:
the convention so far has been to use camel case.
it's your repo, but if you don't stay consistent you'll end up having horrible code.
remember, this isn't python.